app/controllers/user_organization_memberships_controller.rb

```diff
@@ -18,21 +16,11 @@ def organization_memberships
     render json: OrganizationMembershipsRepresenter.new(@memberships).to_json
   end
 
-  def create_by_email
+  def invite_by_email
     if @user.nil?
-      invite_user params[:email]
-      if !@user.valid?
-        render json: @user.errors, status: :unprocessable_entity
-        return
-      end
-    end
-
-    @membership = UserOrganizationMembership.new user: @user, organization: @organization
-    if @membership.save
-      InternalMailer.organization_changed_member_count(@organization, 'added').deliver_later
-      render json: OrganizationMembershipRepresenter.new(@membership).to_json
+      invite_new_user
     else
-      render json: @membership.errors, status: :unprocessable_entity
+      create_new_membership
```

Comment:
If I'm reading this correctly, then this returns either a OrganizationMembershipRepresentor or a UserOrganizationInvitationRepresentor.  That's confusing on the front end.